AutomatedTesting/Gem/PythonTests/AWS/README.md

```diff
@@ -2,30 +2,61 @@
 
 ## Prerequisites
 1. Build the O3DE Editor and AutomatedTesting.GameLauncher in Profile.
-2. AWS CLI is installed and configured following [Configuration and Credential File Settings](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html).
-3. [AWS Cloud Development Kit (CDK)](https://docs.aws.amazon.com/cdk/latest/guide/getting_started.html#getting_started_install) is installed.
+2. Install the latest version of NodeJs.
+3. AWS CLI is installed and configured following [Configuration and Credential File Settings](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html).
+4. [AWS Cloud Development Kit (CDK)](https://docs.aws.amazon.com/cdk/latest/guide/getting_started.html#getting_started_install) is installed.
 
 ## Deploy CDK Applications
 1. Go to the AWS IAM console and create an IAM role called o3de-automation-tests which adds your own account as as a trusted entity and uses the "AdministratorAccess" permissions policy.
-2. Copy {engine_root}\scripts\build\Platform\Windows\deploy_cdk_applications.cmd to your engine root folder.
-3. Open a new Command Prompt window at the engine root and set the following environment variables:
-```
-   Set O3DE_AWS_PROJECT_NAME=AWSAUTO
-   Set O3DE_AWS_DEPLOY_REGION=us-east-1
-   Set O3DE_AWS_DEPLOY_ACCOUNT={your_aws_account_id}
-   Set ASSUME_ROLE_ARN=arn:aws:iam::{your_aws_account_id}:role/o3de-automation-tests
-   Set COMMIT_ID=HEAD
-```
-4. In the same Command Prompt window, Deploy the CDK applications for AWS gems by running deploy_cdk_applications.cmd.
+2. Copy the following deployment script to your engine root folder:
+    * Windows
+        ```
+            {engine_root}\scripts\build\Platform\Windows\deploy_cdk_applications.cmd
+        ```
+    * Linux
+        ```
+            {engine_root}/scripts/build/Platform/Linux/deploy_cdk_applications.sh
+        ```
+3. Open a new CLI window at the engine root and set the following environment variables:
+    * Windows

```

Comment:
suggest specifying "Windows **(command prompt)**", as the syntax for setting env vars in PowerShell (many Windows users' first choice of terminal) is different.

Reply:
Sure. I'll add that.